src/ui/about_dialog.ui

```diff
@@ -12,7 +12,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>780</width>
+    <width>926</width>
```

Comment:
I think that width looks suspiciously large - whilst WinDoze 8.1 has a required minimum resolution of 1024x768, 10 supports as low as 800x600 apparently... :stuck_out_tongue_winking_eye: